README.md

```diff
@@ -104,3 +104,4 @@ x                    | @svenreimers@mastodon.social        | Sven Reimers
 @tomsontom           | @tomsontom@mastodon.social          | Tom Schindl
 @TonyPrintezis       | @TonyPrintezis@mastodon.social      | Tony Printezis
 @trisha_gee          | @trishagee@mastodon.social          | Trisha Gee
+@nicolas_frankel     | @frankel@mastodon.top               | Nicolas Fränkel
```

Comment:
May I ask you to put your name to the alphabetic position of the Twitter handle? Thx!

Reply:
Sure!

I didn't understand the order 😅

Reply:
No problem. Looks like we need to switch to a database backend soon 😅